spec/models/supplemental_file_spec.rb

```diff
@@ -84,4 +90,12 @@
       expect(subject.reload.language).to eq "ger"
     end
   end
+
+  describe '.convert_from_srt' do
+    let(:file) { Rails.root.join('spec', 'fixtures', 'captions.srt') }
+    let(:output) { "WEBVTT\n\n1\n00:00:03.498 --> 00:00:05.000\n- Example Captions" }
+    it 'converts SRT format captions into VTT captions' do
+      expect(SupplementalFile.convert_from_srt(File.read(file))).to eq output
+    end
```

Comment:
What do you think about inlining the SRT file?  I think it makes the test easier to read having everything in one place.
```suggestion
    let(:input) { "1\n00:00:03.498 --> 00:00:05.000\n- Example Captions\n"  }
    let(:output) { "WEBVTT\n\n1\n00:00:03.498 --> 00:00:05.000\n- Example Captions" }
    it 'converts SRT format captions into VTT captions' do
      expect(SupplementalFile.convert_from_srt(input)).to eq output
    end
```